src/custom-forms/use-custom-forms.js

```diff
@@ -1,15 +1,20 @@
 import { useQuery } from 'react-query'
+import { createSelector } from 'reselect'
 import keys from './query-key-factory.js'
 
+const getDataSets = (data) => data.dataSets
+const getCustomForms = createSelector(getDataSets, (dataSets) =>
```

Comment:
Why use `createSelector` here? it's just being used by the hook once data has been retrieved, not in shared hooks